src/eureka/lib/util.py

```diff
@@ -364,6 +364,18 @@ def find_fits(meta):
     return meta
 
 
+def binData(data, nbin=60, err=False):
```

Comment:
Lacks documentation

Reply:
Sorry, this is a draft PR and still in prep - I was going to submit it and then decided to add a plot showing the phase variations in this same PR so I just left this as a draft. I'll mark this as ready for you in a day or two when I'm finished and everything is tested and documented